doc/classes/MissingNode.xml

```diff
@@ -13,8 +13,11 @@
 		<member name="original_class" type="String" setter="set_original_class" getter="get_original_class">
 			The name of the class this node was supposed to be (see [method Object.get_class]).
 		</member>
+		<member name="original_scene" type="String" setter="set_original_scene" getter="get_original_scene">
+			Returns the path of the scene this node was instance of originally.
+		</member>
 		<member name="recording_properties" type="bool" setter="set_recording_properties" getter="is_recording_properties">
-			If set to [code]true[/code], allows new properties to be added on top of the existing ones with [method Object.set].
+			If [code]true[/code], setting a non-existent property will store its value in the node's internal HashMap. If [code]false[/code], only existing properties' values will be accepted.
```

Comment:
Hii, it was me that wrote the prior description. There's no point mentioning the internal HashMap here. It's not something the user can access directly. I would personally revert the prior description a bit (but keep the mention of what happens if `false`).
```suggestion
			If [code]true[/code], allows new properties to be set along with existing ones. If [code]false[/code], only existing properties' values can be set, and new properties cannot be added.
```

Bike-shedding, I know.